valuable-derive/Cargo.toml

```
@@ -1,10 +1,10 @@
 [package]
 name = "valuable-derive"
 version = "0.1.0"
-authors = ["Carl Lerche <me@carllerche.com>", "Taiki Endo <te316e89@gmail.com>"]
 edition = "2018"
 license = "MIT"
-description = "Values"
+rust-version = "1.51.0"
+description = "Macros for `valuable` crate."
 
```

also, we should probably add all the same metadata as `valuable` itself?
```suggestion
repository = "https://github.com/tokio-rs/valuable"
categories = [
    "development-tools::debugging",
    "encoding",
]
keywords = [
    "valuable",
    "serialization",
    "debugging",
    "no_std",
]
```